composer.json

```diff
@@ -2,10 +2,11 @@
     "name": "react/react",
     "description": "ReactPHP: Event-driven, non-blocking I/O with PHP.",
     "keywords": ["reactor", "asynchronous", "ReactPHP"],
+    "homepage": "https://reactphp.org/",
     "license": "MIT",
     "support": {
         "issues": "https://github.com/reactphp/react/issues",
-        "irc": "irc://irc.freenode.org/reactphp"
+        "chat": "https://gitter.im/reactphp/reactphp"
```

Comment:
Should we add this to all our packages?

Reply:
I'm undecided, but I can see some use for this :man_shrugging:

Reply:
It shows up on the packagist page for each package so might not be a bad idea 😄

Reply:
@WyriHaximus My vote would be to postpone this discussion until we see we're happy with adoption of the new chat room and then discuss this in a separate ticket in a few weeks and not hold this one back :shipit:

Reply:
@clue of  course, just thinking out loud :)